charmcraft.yaml

```diff
@@ -0,0 +1,28 @@
+name: github-profiles-automator
+type: charm
+title: GitHub Profiles Automator
+
+summary: A charm for automating the management of Kubeflow Profiles from a GitHub repo
+
+description: |
+  This charm is responsible for monitoring a file from a GitHub repo that represents
+  which Profiles and Contributors should exist in a cluster. Then, via a reconciliation loop
+  the charm will update the Profiles and RoleBindings and AuthorizationPolicies in the
+  cluster to align with this representation.
+
+
+base: ubuntu@24.04
+platforms:
+  amd64:
+
+resources:
+
+parts:
+  charm:
+    charm-python-packages: [setuptools]
+    # https://github.com/canonical/charmcraft/issues/1722
```

Comment:
IIUC it's closed because the solution for now is to have a newer version of `rustc` from the one available in APT. So they closed the issue because it's a problem with `rustc` and not in charmcraft

Reply:
And yep, I was still hitting that issue

Reply:
@kimwnasptd can you update the comment mentioning this context? Having only the link to the issue may be misleading.